src/main/webapp/components/Documents.js

```diff
@@ -144,10 +243,57 @@ export const Documents = hh(class Documents extends Component {
         })
       ]),
 
+      Panel({ title: "Data Use Letter" }, [
```

Comment:
This is what I think should be broken out into a new component.

Reply:
Sounds good, will do